<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dl_Register</class>
 <widget class="QDialog" name="dl_Register">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScorMSG  -  Register</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons/Icons.qrc">
    <normaloff>:/General/App_Icon.png</normaloff>:/General/App_Icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="4">
    <widget class="QPushButton" name="pb_Confirm">
     <property name="text">
      <string>Confirm</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/Icons.qrc">
       <normaloff>:/General/Check.png</normaloff>:/General/Check.png</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pb_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/Icons.qrc">
       <normaloff>:/General/Cancel.png</normaloff>:/General/Cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pb_Login">
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/Icons.qrc">
       <normaloff>:/General/Login.png</normaloff>:/General/Login.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="lb_Message">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Register</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_Password">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_ConfirmPassword">
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_UserID">
        <property name="text">
         <string>User ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_Password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_ShowPassword">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Icons/Icons.qrc">
          <normaloff>:/General/Show_Password.png</normaloff>:/General/Show_Password.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="le_UserID"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="le_ConfirmPassword">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_UserID</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>pb_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
